pages/events.js

```diff
@@ -49,10 +51,10 @@ export default function Events({ events }) {
         {!events.length && <Alert type="info" message="No events found" />}
         <ul role="list" className="divide-y divide-gray-200">
           {categorisedEvents[eventType].map((event, index) => (
-            <EventCard event={event} username={event.username} key={index} />
+            <EventCard event={event} username={event.username} key={event.id} />
```

Comment:
Thank you for the fix 👍 

Could we concatenate the event name with the username?

Reply:
Sure, thanks for the feedback!